vowpalwabbit/parse_args.cc

```diff
@@ -717,8 +717,8 @@ void parse_feature_tweaks(options_i& options, VW::workspace& all, bool interacti
        interactions_settings_duplicated /*settings were restored from model file to file_options and overriden by params from command line*/)
   {
     all.logger.err_warn(
-        "model file has set of {-q, --cubic, --interactions} settings stored, but they'll be "
-        "OVERRIDDEN by set of {-q, --cubic, --interactions} settings from command line.");
+        "model file has set of {{-q, --cubic, --interactions}} settings stored, but they'll be "
+        "OVERRIDDEN by set of {{-q, --cubic, --interactions}} settings from command line.");
```

Comment:
is this tested anywhere?

Reply:
Yes it is hit in one of the integration tests